### PR TITLE
OCP-23543 fixes

### PR DESCRIPTION
**Pull request by anuragthehatter**
fixed OCP-23543 to alleviate STDERR issues on debug pods
@zhaozhanqi http://pastebin.test.redhat.com/824682
PTAL. This will work fine now irrespective of whether we get STDERR or not

### CODE REVIEW

File: features/networking/sdn.feature [hunk 1]
```diff
@@ -299,4 +299,4 @@ Feature: SDN related networking scenarios
       | curl | <%= cb.pod1_ip%>:8080 |
     Then the step should succeed
     And the output should contain "Hello OpenShift"
-    """
+    """
```
Comment (zhaozhanqi) [inline features/networking/sdn.feature]: please check the end of this line. seems there is redundent character

Comment (anuragthehatter) [reply features/networking/sdn.feature]: @zhaozhanqi Seems okay to me. See under "view file"

**Comment by zhaozhanqi** [comment]
need to resolve the conflicts firstly

**Comment by zhaozhanqi** [comment]
@anuragthehatter please fetch the latest code, I saw @weliang1 's case will be deleted by this.

**Pull request merged**